src/components/ImageUploader/README.md

```diff
@@ -7,8 +7,8 @@ Allows JPEG, PNG, and GIF file formats. HEIC is converted to JPEG by iOS with th
 ## Basic Usage With Upload Handler
 The upload hander (passed in as the `uploadHandlerFn` prop) should accept an object as the first argument. This object will contain the fields:
```

Comment:
huh? but it's not called `uploadHandlerFn` it's called `uploadHandler`: https://github.com/square/maker/blob/image-uploader-drag-n-drop/src/components/ImageUploader/src/ImageUploader.vue#L47

please update the doc and/or the code to match